### DISCUSSION TITLE
Kitten 🐾

### DISCUSSION THREAD

**Discussion by rozaxe** [supabase/supabase#374] [Show and tell]
Hi!

I just wanted to tell you that Supabase is really awesome to build upon!

It is so great that I finish building a web-app using it and I wanted to show it to you.

(Also more examples on "how to Supabase" would be great, I think).

So, I built [Kitten](https://kitten-bank.vercel.app/), an open-source app that helps you save money for the things you love.

Basically, it allows you to split your treasury (the money available in your bank's account) into kitties (e.g. Trip to Paris, PS5, Housing, etc.) you wish to save money for.

(No banking information is ever needed, everything is done through the user's manual input.)

You can find the source code [here](https://github.com/rozaxe/kitten).

If you have any suggestions, do not hesitate to tell me!

***

And here a screen of the app (full internationalization coming soon)

![Kitten app](https://user-images.githubusercontent.com/3525604/101982486-8af33680-3c74-11eb-9fc6-00146e8e8e45.png)

**Comment by kiwicopple** [comment]
This is awesome @rozaxe 💯

**Comment by awalias** [comment]
wow this is great 🐾🐾🐾 nice one!

> (Also more examples on "how to Supabase" would be great, I think).

good feedback @rozaxe , any other feedback we'd love to hear it 👍

**Comment by ykdojo** [comment]
It's very cool! I ended up playing around with it a bit.

> (Also more examples on "how to Supabase" would be great, I think).

Thanks for the feedback, and please let us know if you'd like to see any specific type of examples :)
